### PR TITLE
chore: update readme to include Accessibility Insights for Android canary/insider links

### PR DESCRIPTION
**Pull request by karanbirsingh**
#### Description of changes

We don't list the canary / insider links anywhere yet. This PR includes them. See what it looks like [here](https://github.com/microsoft/accessibility-insights-web/blob/65ce5f4631a85ed859d50c4a030d57111b423703/README.md).

#### Pull request checklist

- [ ] Addresses an existing issue: #0000
- [ ] Ran `yarn fastpass`
- [ ] Added/updated relevant unit test(s) (and ran `yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [ ] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). Check workflow guide at: `<rootDir>/docs/workflow.md`
- [ ] (UI changes only) Added screenshots/GIFs to description above
- [ ] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,16 +12,33 @@ Licensed under the MIT License.
...
 
 -   ![Canary Logo](./src/icons/brand/red/brand-red-16px.png) [Canary](https://chrome.google.com/webstore/detail/hbcplehnakffdldhldncjlnbpfgogbem) (released continuously)
 -   ![Insider Logo](./src/icons/brand/violet/brand-violet-16px.png) [Insider](https://chrome.google.com/webstore/detail/nnmjfbmebeckhpejobgjjjnchlljiagp) (on feature completion)
 -   ![Production Logo](./src/icons/brand/blue/brand-blue-16px.png) [Production](https://chrome.google.com/webstore/detail/pbjjkligggfmakdaogkfomddhfmpjeni) (after validation in Insider)
 
+### Running Accessibility Insights for Android
+
+You can install Accessibility Insights for Android from one of the following links. The application requires an APK which you can download [here](https://accessibilityinsights.io/en/downloads).
+
+-   MacOS ([Canary](https://aka.ms/accessibility-insights-for-android/downloads/CanaryMacOS), [Insider](https://aka.ms/accessibility-insights-for-android/downloads/InsiderMacOS), [Production](https://aka.ms/accessibility-insights-for-android/downloads/MacOS))
```
Comment (karanbirsingh) [inline README.md]: @ferBonnin I have added two duplicate formats, one with bullets and one with a table. Which do you like better? You can see it [here](https://github.com/microsoft/accessibility-insights-web/blob/1e5d423653196b7c00dce2737d4fe9ac67ff7533/README.md).

Comment (ferBonnin) [reply README.md]: thanks! I like the first one, bullet points better: visually, it looks cleaner

File: README.md [hunk 2]
```diff
@@ -12,16 +12,33 @@ Licensed under the MIT License.
...
 [![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=microsoft/accessibility-insights-web)](https://dependabot.com)
 
-Accessibility Insights for Web is a browser extension for Google Chrome and the new Microsoft Edge, used for assessing the accessibility of web sites and web applications.
+Two projects are built from this repository:
 
-### Running the extension
+-   Accessibility Insights for Web is a browser extension for Google Chrome and the new Microsoft Edge, used for assessing the accessibility of web sites and web applications.
+-   Accessibility Insights for Android is a cross-platform desktop tool used for testing accessibility of Android applications.
+
+### Running Accessibility Insights for Web
```
Comment (smoralesd) [inline README.md]: should this be `Installing` instead of `Running?`

File: README.md [hunk 3]
```diff
@@ -12,16 +12,25 @@ Licensed under the MIT License.
 [![Chrome Web Store](https://img.shields.io/chrome-web-store/stars/pbjjkligggfmakdaogkfomddhfmpjeni.svg)](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni/reviews)
 [![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=microsoft/accessibility-insights-web)](https://dependabot.com)
 
-Accessibility Insights for Web is a browser extension for Google Chrome and the new Microsoft Edge, used for assessing the accessibility of web sites and web applications.
+Two projects are built from this repository:
 
-### Running the extension
+-   Accessibility Insights for Web is a browser extension for Google Chrome and the new Microsoft Edge, used for assessing the accessibility of web sites and web applications.
```
Comment (ferBonnin) [inline README.md]: recommend making the name of the two products in this bulleted lists bold

**Pull request merged**